### PR TITLE
VS Friends typo fixes + change wave animation from cubic-bezier to linear

### PR DESCRIPTION
**Pull request by vixspade**
VS Friends: Isa's "not the pranks" is wavy instead of shaky, source: https://www.youtube.com/live/VnPyUhcPhgk?si=sJIEf6IvOE7SCTua&t=11440

the changing of the wave is to make it more accurate to the game, although it might break for Mysterious Reasons

### CODE REVIEW

File: public/bars.js [hunk 1]
```diff
@@ -304,7 +304,9 @@ function applyWaveAnimation() {
                 children[child].classList.remove("wave-visual");
     
                 children[child].style.display = "inline-block";
-                children[child].style.animation = `${waveSpeed}s wave cubic-bezier(0.37, 0, 0.63, 1) infinite ${-child * wavelength}s alternate`;
+                // if live editing a page to change all cubic-bezier to linear this looks weird but hopefully it works fine
+                // linear is more accurate to the game (with an amplitude of 15-20ish % ?
+                children[child].style.animation = `${waveSpeed}s wave linear(0.37, 0, 0.63, 1) infinite ${-child * wavelength}s alternate`;
```
Comment (goldstargloww) [inline public/bars.js:309]: firefox_4nwWLUk7Bo

not sure exactly what you were going for so i can't fix it but i don't think this was it lol

**Review by goldstargloww — CHANGES_REQUESTED** [review]

File: public/bars.js (hunk 1 above)
Comment (vixspade) [reply public/bars.js]: oops the numbers in the brackets do mean something actually. should be fixed in commit [1e1649a](https://github.com/felikatze/isat-script-project/pull/226/commits/1e1649a3323c63a3cc0ee7f60d8c4ec3bb7f8e42)?

Comment (goldstargloww) [reply public/bars.js]: yep, looks good! in the cubic-bezier version the numbers are the xy coordinates of the two handles, but in the linear version they seemed to be where the points were at a given point

**Review by goldstargloww — APPROVED** [review]

**Review by goldstargloww — APPROVED** [review]

**Comment by goldstargloww** [comment]
thanks :D

**Pull request merged**